flutter_loggy_dio/lib/flutter_loggy_dio/dio_logger.dart

```diff
@@ -164,9 +153,9 @@ class LoggyDioInterceptor extends Interceptor with DioLoggy {
 
   void _commit(LogLevel level) {
     if (level.priority >= LogLevel.error.priority) {
-      final _valueError = _value.toString();
-      final _errorTitle = _valueError.substring(0, _valueError.indexOf('\n'));
-      final _errorBody = _valueError.substring(_errorTitle.length);
+      final String _valueError = _value.toString();
+      final String _errorTitle = _valueError.substring(0, _valueError.indexOf('\n'));
+      final String _errorBody = _valueError.substring(_errorTitle.length);
```

Comment:
```suggestion
      final _valueError = _value.toString();
      final _errorTitle = _valueError.substring(0, _valueError.indexOf('\n'));
      final _errorBody = _valueError.substring(_errorTitle.length);
```
This was ok, as it was 😄

Reply:
We have `pedantic` as analysis options in here, and analysis is not happy with previous solution 😄